test/metabase/automagic_dashboards/core_test.clj

```diff
@@ -918,13 +917,15 @@
                        Field    _ {:table_id table-id}
                        Field    _ {:table_id table-id}]
           (automagic-dashboards.test/with-dashboard-cleanup
-            (is (schema= [(s/one {:tables   [(s/one {:table    {:id       (s/eq table-id)
-                                                                s/Keyword s/Any}
-                                                     s/Keyword s/Any}
-                                                    "first Table")]
-                                  s/Keyword s/Any}
-                                 "first result")]
-                         (magic/candidate-tables (t2/select-one Database :id db-id))))))))))
+            (is (malli= [:sequential
+                         {:min 1, :max 1}
+                         [:map
+                          [:tables [:sequential
+                                    {:min 1, :max 1}
+                                    [:map
+                                     [:table [:map
+                                              [:id [:= table-id]]]]]]]]]
+                        (magic/candidate-tables (t2/select-one Database :id db-id))))))))))
```

Comment:
This could use `=?` instead
```suggestion
            (is (=? [{:tables [{:table {:id (s/eq table-id)}}]}]
                    (magic/candidate-tables (t2/select-one Database :id db-id))))))))))
```

Reply:
oh yeah, thanks. Fixed